stages/org.osbuild.ostree.deploy.container

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python3
+"""
+Deploy an OStree commit
+
+Create an OSTree deployment[1] for a given container image input
+"""
+
+import os
+import sys
+
+import osbuild.api
+from osbuild.util import containers, ostree
+from osbuild.util.mnt import MountGuard
+
+CAPABILITIES = ["CAP_MAC_ADMIN"]
+
+SCHEMA_2 = """
+"options": {
+  "additionalProperties": false,
+  "required": ["osname", "target_imgref"],
+  "properties": {
+    "mounts": {
+      "description": "Mount points of the final file system",
+      "type": "array",
+      "items": {
+        "description": "Description of one mount point",
+        "type": "string"
+      }
+    },
+    "osname": {
+      "description": "Name of the stateroot to be used in the deployment",
+      "type": "string"
+    },
+    "kernel_opts": {
+      "description": "Additional kernel command line options",
+      "type": "array",
+      "items": {
+        "description": "A single kernel command line option",
+        "type": "string"
+      }
+    },
+    "target_imgref": {
+      "description": "imageref used as the source of truth for updates",
+      "type": "string"
+    },
```

Comment:
Since we don't do any validation on this and the string format might not be immediately obvious from the description, we could add a few examples here using `"examples": [ ... ]` (see other stage schemas that provide examples for options).
We could also add a regex pattern that validates the first component:
```
"pattern": "^(ostree-unverified-image|ostree-unverified-registry|ostree-remote-image|ostree-remote-image|ostree-remote-registry|ostree-image-signed):.*$"
```

Reply:
Should be done now.